include/vcpkg/base/messages.h

```diff
@@ -691,6 +699,22 @@ namespace vcpkg
                     (),
                     "This message is normally displayed only in development.",
                     "Downloading latest vcpkg-ce bundle...");
+    DECLARE_MESSAGE(DownloadRootsDir,
```

Comment:
Environment vars shouldn’t get single quotes.

Note to self: double check that all environment vars are going through their formatter so they get % or $ as appropriate

Reply:
All in vcpkgarguments.cpp or all localized env_var?

Reply:
(I believe this is why {env_var} exists as a thing in the first place rather than just including the name of the variable in the text)